resources/datomic/schema.dtm

```diff
@@ -522,34 +552,100 @@
   :db/ident       :stats/wood-barriers-destroyed
   :db/valueType   :db.type/long
   :db/cardinality :db.cardinality/one
-  :db/doc         "Number of wood-barriers destroyed"}
+  :db/doc         "Number of wood barriers destroyed"}
```

Comment:
Is this for barriers they destroy by ramming through them? (I see `wood-barriers-shot` below, so just guessing .... )